src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -1380,12 +1379,13 @@ static void Main(string[] args)
 @"
 {
         int y,
+#if true
+
```

Comment:
I can live with this behavior.  Critical part is not leaving dangling preprocessor directives.